### DISCUSSION TITLE
Github Actions Aren't Appearing in the Actions tab

### DISCUSSION THREAD

**Discussion by AlexMachin1997** [community/community#25397] [Actions]
<p>Hi Everyone,</p>
<p>So this afternoon I discovered Github Actions and decided to setup a basic action to run my tests when I either push to the master branch or push to any development branch.</p>
<p>However, even though I’ve set my action up I can’t see to find the action nor has it been triggered for the first time.</p>
<p>Im not entirely sure what Im doing wrong, any help would be greatly appreciated :slight_smile:</p>
<p>Repo link:<br>
</p><aside class="onebox whitelistedgeneric">
  <header class="source">
      <img src="https://github.githubassets.com/favicons/favicon.svg" class="site-icon" width="32" height="32">
      <a href="https://github.com/AlexMachin1997/TheOpenMovieDB-React-Example" target="_blank" rel="nofollow noopener">GitHub</a>
  </header>
  <article class="onebox-body">
    <img src="https://user-images.githubusercontent.com/29917195/181090750-dc5be38e-a4df-469f-be78-fd170c427c41.jpeg" class="thumbnail onebox-avatar" width="400" height="400">

<h3><a href="https://github.com/AlexMachin1997/TheOpenMovieDB-React-Example" target="_blank" rel="nofollow noopener">AlexMachin1997/TheOpenMovieDB-React-Example</a></h3>

<p>A Progressive Web Application clone of The Open Movie Database, it using predominan JavaScript based tools like React, Styled-Components, Storybook just to name a fewtly  - AlexMachin1997/TheOpenMo...</p>

  </article>
  <div class="onebox-metadata">
    
    
  </div>
  <div style="clear: both"></div>
</aside>

**Comment by airtower-luna** [comment, marked as answer]
<p>Your workflow file excludes the “master” branch for both push and pull_request events, while master is the only branch in your repository. I assume that’s why the workflow isn’t running. :wink:</p>

**Comment by AlexMachin1997** [comment]
<p>Oh son of a gun, I forgot I added that :laughing:</p>
<p>I have one more question, how can I refer to package scripts inside the .yml file or do I just need to copy and paste the scripts into the file.</p>

**Comment by Yanjingzhu** [comment]
<p>I checked your repo, it is a npm package. In your workflow, after checking out the source code,<br>
you could just use <code>npm run &lt;name&gt;</code> command to run the scripts specified in <code>package.json</code> .</p>
